test/Models/Tab.spec.js

```diff
@@ -2,25 +2,28 @@ const { test, expect, beforeAll, beforeEach } = require('@jest/globals');
 const { DatabaseError } = require('sequelize');
 
 const {
-  models: { Tab, TabDrink },
+  models: { Drink, Tab, TabDrink },
 } = require('../../server/db/index.js');
 const Venue = require('../../server/db/models/Venue.js');
 
-beforeAll(async () => {
-  await Tab.create({ subTotal: 10 });
-  await TabDrink.create({ price: 8.0, quantity: 1 });
-});
-
 describe('Tab Attributes', () => {
   let tab;
   let drink;
-  beforeEach(async () => {
-    tab = await Tab.findOne();
-    drink = await TabDrink.findOne();
-    drink.tabId = tab.id;
-    await drink.save();
+  beforeAll(async () => {
+    tab = await Tab.create({ subTotal: 10 });
+    drink = await Drink.findOne();
+    await tab.addDrink(drink, { through: { price: 8.0, quantity: 1 } });
     await tab.getSubTotal();
   });
```

Comment:
This fixed most of the tests, I believe. So, instead of created a `TabDrink` you want to simply add a `Drink` to a `Tab`; you can use magic methods in Sequelize to do so. Here I call `tab.addDrink(drink)` to set up the association and pass in `through` options for `price` and `quantity` that we'll need later.